PennyPincher/PennyPincherAndroidGesturesImporter.swift

```diff
@@ -134,14 +111,14 @@ public final class PennyPincherAndroidGesturesImporter {
     }
 }
 
-private protocol ByteSwappable {
-    var byteSwapped: Self { get }
+private protocol BigEndian {
```

Comment:
In Swift 3 parlance I think this is perhaps `BigEndianConvertible`, like `ArrayLiteralConvertible`.